sklearn/linear_model/tests/test_ridge.py

```diff
@@ -598,6 +598,34 @@ def test_ridgecv_store_cv_values():
     assert_equal(r.cv_values_.shape, (n_samples, n_responses, n_alphas))
 
 
+def test_ridge_classifier_cv_store_cv_values():
+    # Test RidgeClassifier
+    x = np.array([[-1.0, -1.0], [-1.0, 0], [-.8, -1.0],
+                  [1.0, 1.0], [1.0, 0.0]])
+    y = np.array([1, 1, 1, -1, -1]).reshape(-1, 1)
+
+    n_samples = x.shape[0]
+
+    alphas = [1e-1, 1e0, 1e1]
+    n_alphas = len(alphas)
+
+    r = RidgeClassifierCV(alphas=alphas, store_cv_values=True)
+
+    # with len(y.shape) == 1
+    r.fit(x, y)
+    n_responses = y.shape[1]
```

Comment:
we would usually call these "outputs" or "targets" rather than "responses".

Reply:
Thought the same, but followed nomenclature in the test for RidgeCV in Line 577. I'll change it to target.

Reply:
BTW line 577(I don't know how to reference it from here) seems to have a typo (double assignation). Should I change it?

Reply:
BTW line 579 (I don't know how to reference it from here) seems to have a typo (double assignation). Should I change it?

`rng = rng = np.random.RandomState(42)`